include/vkgcDefs.h

```diff
@@ -200,6 +200,14 @@ enum ShaderStageBit : unsigned {
   ShaderStageComputeBit = (1 << ShaderStageCompute),         ///< Compute shader bit
 };
 
+/// Enumerates LLPC types of unlinked shader elf.
+enum UnlinkedShaderStage : unsigned {
```

Comment:
I guess you might have considered using an enum class here if you had Jakub's enum iterator thing to avoid all the casting in "for" loops.

Reply:
Yes I would have.